src/solver/ts-generator/availability.cpp

```diff
@@ -73,8 +74,9 @@ class GeneratorTempData final
 {
 public:
     explicit GeneratorTempData(Data::Study&, unsigned, MersenneTwister&);
+    explicit GeneratorTempData(bool, unsigned, MersenneTwister&);
 
-    void generateTS(const Data::Area& area, AvailabilityTSGeneratorData& cluster) const;
+    void generateTS(std::string, AvailabilityTSGeneratorData&) const;
```

Comment:
```suggestion
    void generateTS(const std::string&, AvailabilityTSGeneratorData&) const;
```

Reply:
I simply removed this **std::string**  argument : 
indeed, this source area name is used inside function **generateTS** in case **prepro** was not properly loaded.
But the **prepro** loading itself already checks it was loaded correctly and raises an error in case it was not.
So I don't see the point of making this check again when generating the TS.